jsk_2020_10_semi/scripts/auto-temperature-checker.l

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env roseus
+
+(require "package://roseus_smach/src/state-machine-ros.l")
+(require "package://roseus_smach/src/state-machine-utils.l")
+(require "package://fetcheus/fetch-interface.l")
+(require "package://jsk_fetch_startup/euslisp/navigation-utils.l")
+
+(ros::roseus "auto_temperature_checker")
+(ros::roseus-add-msgs "opencv_apps")
+(fetch-init)
+
+(setq *roi-msg* (instance opencv_apps::FaceArrayStamped :init :header (instance std_msgs::Header :init :stamp (ros::time-now))))
+(defun face-roi-cb (msg)
+  (setq *roi-msg* msg))
+
+(setq *name-msg* (instance opencv_apps::FaceArrayStamped :init :header (instance std_msgs::Header :init :stamp (ros::time-now))))
+(defun face-name-cb (msg)
+  (setq *name-msg* msg))
+
+(setq *battery-msg* (instance power_msgs::BatteryState :init :is_charging t))
+(setq *battery-msg-arrived* (ros::time-now))
+(defun battery-state-cb (msg)
+  (setq *battery-msg-arrived* (ros::time-now))
+  (setq *battery-msg* msg))
+
+(ros::subscribe "head_camera/rgb/face_roi" opencv_apps::FaceArrayStamped #'face-roi-cb)
+(ros::subscribe "head_camera/rgb/face_name" opencv_apps::FaceArrayStamped #'face-name-cb)
+(ros::subscribe "battery_state" power_msgs::BatteryState #'battery-state-cb)
+
+(setq *count* 0)
+(defun go-start (userdata-alist)
+  (print "go-start")
+  (auto-undock)
+  (send *ri* :go-pos-unsafe 0 0 180)
+  (let ((pos (car (get-spot-coords "/eng2/7f/room73B2-center"))))
+    (go-to-spot "/eng2/7f/room73B2-center"
```

Comment:
@708yamaguchi `go-to-spot` in `navigation-utils.l` takes `relative-coords`, but it is calculated as
```
      (setq goal-pose (send goal-pose :transform relative-coords :world)))
```
https://github.com/jsk-ros-pkg/jsk_robot/blob/25ac99ab82539b0e95d20470db1675b5f2f58c56/jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l#L106

I just want to change the direction of robot relative to "/eng2/7f/room73B2-center", but have to write something like this, is this what you intended?